engine/src/flutter/shell/platform/windows/flutter_window.cc

```diff
@@ -305,6 +308,22 @@ PointerLocation FlutterWindow::GetPrimaryPointerLocation() {
   return {(size_t)point.x, (size_t)point.y};
 }
 
+FlutterEngineDisplay FlutterWindow::GetDisplay() {
+  HMONITOR monitor =
+      MonitorFromWindow(GetWindowHandle(), MONITOR_DEFAULTTONEAREST);
+  for (auto const& display : display_monitor_->GetDisplays()) {
+    if (display.display_id ==
+        reinterpret_cast<FlutterEngineDisplayId>(monitor)) {
+      return display;
+    }
+  }
+
+  FML_LOG(ERROR) << "This window is not associated with any monitor";
```

Comment:
Just curious, if this happens would it be because Flutter Windows has a bug? If so, should we just crash?

Reply:
On second thought, what do you think of:

1. Changing this method to return just a `FlutterEngineDisplayId`
2. Add a `std::optional<FlutterEngineDisplay> DisplayMonitor::FindById(FlutterEngineDisplayId id)` method
3. In debug mode, use the display monitor to verify the display ID is a known value. If it is not, report the error message

I think that makes the fallback case a little less funky.